language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/resolvers/parsercontext/ParserRuleVariableDefinitionStatementContextResolver.java

```diff
@@ -53,21 +55,18 @@
             ArrayList<SymbolInfo> actionAndFunctions = new ArrayList<>();
             actionAndFunctions.addAll(actionFunctionTypeFilter.filterItems(completionContext));
             this.populateCompletionItemList(actionAndFunctions, completionItems);
-            return completionItems;
         } else {
             // Fill completions if user is writing a connector init
             List<SymbolInfo> filteredConnectorInitSuggestions = connectorInitItemFilter.filterItems(completionContext);
             if (!filteredConnectorInitSuggestions.isEmpty()) {
                 populateCompletionItemList(filteredConnectorInitSuggestions, completionItems);
-                return completionItems;
             }
 
             // Add the create keyword
             CompletionItem createKeyword = new CompletionItem();
             createKeyword.setInsertText("create ");
             createKeyword.setLabel("create");
```

Comment:
can we create a const class to keep keywords like create, var which are used in few places?

Reply:
Added a constant as per suggestion